test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -30,6 +30,8 @@
 
 apply plugin: 'opensearch.java'
 
+group = 'hdfs'
```

Comment:
Is the name of the group fine? The above 2 names are `fixture.xxx`, but these 2 names without the `fixture` prefix.

Reply:
Yeah, that discrepancy exists from the legacy codebase. The imports inside hdfs module use package name as `hdfs` unlike other fixture modules. Probably we can fix the hdfs module in a separate PR.